app/metrics.yaml

```diff
@@ -1249,3 +1249,29 @@ context_menu:
           open_in_new_tab, open_in_private_tab, open_image_in_new_tab,
           save_image, share_link, copy_link, copy_image_location, share_image
         type: string
+
+search_suggestions:
+  suggestion_tapped:
+    type: event
+    description: Search suggestion selected.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5662
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/?
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-11-01"
```

Comment:
@mcarare I think we want the same `engine_name` extra here that we have in `search_bar.performed_search` right?

```
    extra_keys:
      engine_name:
        description: The name of the engine used to perform the search.
        type: string
```

Reply:
Or does this trigger a search event later that does have the engine_name in the event? If that is the case then this is probably fine.

Reply:
It does not trigger the other event, as that is specific to another action. 
I added the engine name as an extra key.

Reply:
I'll do the same on iOS.